columnflow/columnar_util.py

```diff
@@ -366,6 +366,15 @@ def __eq__(self, other: Route | str | Sequence[str | int | slice | type(Ellipsis
             return self.column == other
         return False
 
+    def __lt__(self, other: Route | str | Sequence[str | int | slice | type(Ellipsis) | list]) -> bool:
```

Comment:
is this used anywhere?

Reply:
It's needed when sorting sequences of Route objects. We already have `__eq__`, so `__lt__` was the only missing piece.